index.d.ts

```diff
@@ -0,0 +1,519 @@
+declare const RadioBrowser: {
+    service_url: string;
+
+    /**
+     * get a radio-browser host
+     */
+    getRandomHost(): string;
+    
+    /**
+     * returns a list of category.
+     * https://de1.api.radio-browser.info/#List_of_countries
+     * https://de1.api.radio-browser.info/#List_of_countrycodes
+     * https://de1.api.radio-browser.info/#List_of_codecs
+     * https://de1.api.radio-browser.info/#List_of_languages
+     * https://de1.api.radio-browser.info/#List_of_states
+     * https://de1.api.radio-browser.info/#List_of_tags
+     */
+    getCategory<T extends radioBrowser.Categories>(category: T, filter?: Partial<radioBrowser.CategoryFilter>): Promise<radioBrowser.Category<T>[]>;
+    
+    /**
+     * List of radio stations
+     * https://de1.api.radio-browser.info/#List_of_radio_stations
+     * https://de1.api.radio-browser.info/#Search_radio_stations_by_url
+     * https://de1.api.radio-browser.info/#Stations_by_clicks
+     * https://de1.api.radio-browser.info/#Stations_by_votes
+     * https://de1.api.radio-browser.info/#Stations_by_recent_click
+     * https://de1.api.radio-browser.info/#Stations_by_recently_changed
+     * https://de1.api.radio-browser.info/#Stations_that_got_deleted
+     * https://de1.api.radio-browser.info/#Old_versions_of_stations
+     * https://de1.api.radio-browser.info/#Stations_that_need_improvement
+     * https://de1.api.radio-browser.info/#Broken_stations
+     */
+    getStations(filter?: Partial<radioBrowser.StationsFilter>): Promise<Partial<radioBrowser.StationStruct[]>>
+    
+    /**
+     * Get a list of station check results
+     * https://de1.api.radio-browser.info/#Get_a_list_of_station_check_results
+     * 
+     * @default seconds 0
+     */
+    getChecks(stationuuid: string, seconds?: number): Promise<Partial<radioBrowser.StationCheckStruct>>
+    
+    /**
+     * List of station check steps
+     * https://de1.api.radio-browser.info/#List_of_station_check_steps
+     * 
+     * @param {array} uuids array of station uuids
+     */
+    getChecksteps(uuids: (string|number)[]): Promise<Partial<radioBrowser.StationCheckStepStruct>>
+    
+    /**
+     * List of station clicks
+     * <https://de1.api.radio-browser.info/#List_of_station_clicks>
+     * @default seconds 0
+     */
+    getClicks(stationuuid: string, seconds?: number): PromiseradioBrowser.StationClicks>
```

Comment:
PromiseradioBrowser.StationClicks>
needs to be
Promise<radioBrowser.StationClicks>